### PR TITLE
feat: zookeeper client support kerberos

### PR DESCRIPTION
**Pull request by GehaFearless**
issue: https://github.com/apache/incubator-pegasus/issues/847

### CODE REVIEW

**Review by acelyc111 — APPROVED** [review]
LGTM

File: .github/workflows/ci-pull-request.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
     needs: lint
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/pegasus-kv/thirdparties-bin:ubuntu1804
+      image: apachepegasus/thirdparties-bin:ubuntu1804
```
Comment (foreverneverer) [inline .github/workflows/ci-pull-request.yaml:66]: has it updated to `apapche/pegasus`?

File: scripts/pack_server.sh [hunk 2]
```diff
@@ -105,13 +105,6 @@ copy_file ./rdsn/thirdparty/output/lib/libPoco*.so.* ${pack}/bin
 copy_file ./rdsn/thirdparty/output/lib/libtcmalloc_and_profiler.so.4 ${pack}/bin
 copy_file ./rdsn/thirdparty/output/lib/libboost*.so.1.69.0 ${pack}/bin
 copy_file ./rdsn/thirdparty/output/lib/libhdfs* ${pack}/bin
-copy_file ./rdsn/thirdparty/output/lib/libsasl2.so.3 ${pack}/bin
-copy_file ./rdsn/thirdparty/output/lib/libcom_err.so.3 ${pack}/bin
-copy_file ./rdsn/thirdparty/output/lib/libgssapi_krb5.so.2 ${pack}/bin
-copy_file ./rdsn/thirdparty/output/lib/libkrb5support.so.0 ${pack}/bin
-copy_file ./rdsn/thirdparty/output/lib/libkrb5.so.3 ${pack}/bin
-copy_file ./rdsn/thirdparty/output/lib/libk5crypto.so.3 ${pack}/bin
-copy_file ./rdsn/thirdparty/output/lib/sasl2 ${pack}/bin
```
Comment (foreverneverer) [inline scripts/pack_server.sh:114]: why delete?

Comment (GehaFearless) [reply scripts/pack_server.sh:114]: as discuss before, libkrb5-dev libsasl2-dev install by yum，so I do it

Comment (foreverneverer) [reply scripts/pack_server.sh:114]: Yeah, but actually I haven't understand why this?

File: .github/workflows/ci-pull-request.yaml (hunk 1 above)
Comment (acelyc111) [reply .github/workflows/ci-pull-request.yaml:66]: I'll do it in https://github.com/apache/incubator-pegasus/pull/949, this patch doesn't have to do this work.

File: scripts/pack_server.sh (hunk 2 above)
Comment (GehaFearless) [reply scripts/pack_server.sh:114]: Those thirdparties not exist on env. To be consistent, I need to find and copy those from system lib.

**Review by foreverneverer — APPROVED** [review]

**Pull request merged**